lib/galaxy/config/__init__.py

```diff
@@ -1103,10 +1104,36 @@ def _process_config(self, kwargs: Dict[str, Any]) -> None:
         self.proxy_session_map = self.dynamic_proxy_session_map
         self.manage_dynamic_proxy = self.dynamic_proxy_manage  # Set to false if being launched externally
 
-        # InteractiveTools propagator mapping file
-        self.interactivetools_map = self._in_root_dir(
+        # InteractiveTools propagator mapping
+        self.interactivetools_map = "sqlite:///" + self._in_root_dir(
```

Comment:
Given that `interactivetools_map` is ignored if `interactivetools_map_sqlalchemy` is set, I think we should *first* check `interactivetools_map_sqlalchemy`. If it's not set, then build the sqlite url; but if it is set, then set `interactivetools_map` to None for clarity (and to avoid potential bugs in the future)

Reply:
Addressed in 26fd210c